### PR TITLE
-XX:+TransparentHugePage enabled by default on plinux and zlinux

### LINKED ISSUE #421: -XX:+TransparentHugePage is enabled by default on plinux and zlinux
**Issue or pull request number:**
https://github.com/eclipse/openj9/pull/7665

**Overview:**
-XX:+TransparentHugePage is enabled by default on plinux and zlinux

**Release target:**
OpenJ9 0.18

**Applies to the following JDK versions:**
all

**Applies to the following platforms:**
 plinux and zlinux

### PR DESCRIPTION
**Pull request by pshipton**
Closes #421

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by DanHeidinga** [comment]
@SueChaplain can you review as well?

File: docs/version0.18.md [hunk 1]
```diff
@@ -47,6 +48,10 @@ You can set the `-Xshareclasses:noTimestampChecks` option to turn off timestamp
 
 On z/OS 64-bit, [`-Xmso`](xmso.md) has a 1 MB minimum value, to match the minimum stack space provided by the operating system. If you set a value smaller than 1 MB, the value is ignored.
 
+### -XX:+TransparentHugePage is enabled by default on Linux&reg POWER&reg and IBM Z&reg systems
+
+On Linux systems for POWER and IBM Z, [-XX:+TransparentHugePage](xxtransparenthugepage.md) is enabled by default. This option only takes affect when Transparent Huge Pages (THP) is set to `madvise` on your system. When transparent huge pages are used, application footprint may increase.
```
Comment (SueChaplain) [inline docs/version0.18.md]: Misplaced "only": This option takes affect only when .....
Replace "may" with "might".

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/version0.18.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ The following new features and notable changes since v 0.17.0 are included in th
 - [jextract now available on macOS for OpenJDK version 8](#jextract-now-available-on-macos-for-openjdk-version-8)
 - [New shared classes cache suboption to turn off timestamp checking](#new-shared-classes-cache-suboption-to-turn-off-timestamp-checking)
 - [`-Xmso` 1 MB minimum value on z/OS 64-bit](#-xmso-1-mb-minimum-value-on-zos-64-bit)
+- [-XX:+TransparentHugePage is enabled by default on Linux&reg POWER&reg and IBM Z&reg systems](#-XXTransparentHugePage-is-enabled-by-default-on-Linux-POWER-and-IBM-Z-systems)
```
Comment (SueChaplain) [inline docs/version0.18.md]: Linux is TM, use `&trade;`
Registered trademarks missing the closing ";" i.e.`&reg;`

Comment (pshipton) [reply docs/version0.18.md]: Fixed.

File: docs/version0.18.md (hunk 1 above)
Comment (pshipton) [reply docs/version0.18.md]: Fixed.

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

**Pull request merged**